pkg/cmd/get/get_pipeline.go

```diff
@@ -176,7 +180,8 @@ func (o *GetPipelineOptions) dump(jenkinsClient gojenkins.JenkinsClient, name st
 
 	if job.Jobs != nil {
 		for _, child := range job.Jobs {
-			o.dump(jenkinsClient, job.FullName+"/"+child.Name, table)
+			//Todo: Recursive call
+			o.dump(jenkinsClient, job.FullName+"/"+child.Name, table) //nolint:errcheck
```

Comment:
I guess the idea is to find child jobs !?

I am wondering though whether `get pipelines` is not Static Master specific anyways and hence obsolete. My guess is we should delete the whole command.

@abayer - wdyt?